runtime/interpreter/stringatreevalue.go

```diff
@@ -28,6 +28,7 @@ type StringAtreeValue string
 
 var _ atree.Value = StringAtreeValue("")
 var _ atree.Storable = StringAtreeValue("")
+var _ atree.ComparableStorable = StringAtreeValue("")
```

Comment:
Is this also needed for `Uint64AtreeValue`?

Reply:
Unless I'm mistaken, it probably isn't needed because:
- `atree.ComparableStorable` is only needed for `atree.OrderedMap` keys in Cadence `CompositeValue`
- `StringAtreeValue` seems to be the only key type used by `CompositeValue`

Reply:
`StringAtreeValue` is also used for "non-`interpreter.Value`" storage, i.e. for the atree ordered maps used for account storage, and `Uint64AtreeValue` is only used for those.

For example, saving a value to `/storage/foo` actually is implemented as storing a value inside of the account storage's ordered map for the `storage` domain, under key `StringAtreeValue("foo")`

Reply:
This was discussed in today's meeting and it isn't for `Uint64AtreeValue` needed as mentioned above.